tests/bootstrap.php

```diff
@@ -37,5 +37,22 @@
 	'active_plugins' => array( basename( TESTS_PLUGIN_DIR ) . '/load.php' ),
 );
 
+// Add filter to ensure the plugin's admin integration and all modules are loaded for tests.
+require_once $_test_root . '/includes/functions.php';
+require_once TESTS_PLUGIN_DIR . '/tests/testdata/class-wp-image-supports-webp.php';
```

Comment:
Hmm, what about the other new test class file - doesn't that need to be `require once`d here too?

Also, can you move this line somewhere above or below this new code section here, since it doesn't have to do with the rest that is happening here, specifically not the comment above.